modules/core/src/main/java/org/apache/ignite/internal/processors/bulkload/BulkLoadProcessor.java

```diff
@@ -91,15 +103,17 @@ public BulkLoadCacheWriter outputStreamer() {
      * @throws IgniteIllegalStateException when called after {@link #close()}.
      */
     public void processBatch(byte[] batchData, boolean isLastBatch) throws IgniteCheckedException {
```

Comment:
Did you consider adding separate span for batch processing and using qrySpan as parentOne?